test-dataset-plan.md

```diff
@@ -0,0 +1,39 @@
+# Plan for the test dataset
+
+We want the test dataset to contain an equal amount of data split into various
+tasks, described below.
+
+Aside from ensuring that the resulting models perform well across multiple
+tasks and do not overfit to any single one such, this would also provide
+transparent performance descriptions of the trained models. Such transparency
+would mean that anyone who is looking for a language model to finetune on their
+dataset can immediately see whether the model works well on their domain/task.
+
+## Tasks
+
+As mentioned above, we want the test dataset to contain an equal amount of
+data from each of the following tasks:
+
+- Danish web data (Netarkivet)
+- Danish social media data (HOPE Twitter, Reddit-da and Hestenettet from Danish
+  Gigaword)
+- Danish legal data (Retsinformation and Skat.dk, both from Danish Gigaword)
+- Danish wiki data (Danish Wikipedia and lex.dk)
+- Danish news data (DaNews)
+- Danish books (Gutenberg, Danish literature, WikiBooks and WikiSource, all
+  from Danish Gigaword)
+- Danish spontaneous text (OpenSubtitles and Spontaneous Speech, both from
+  Danish Gigaword)
+- Danish dialectal data (Botxt and Sønderjysk, both from Danish Gigaword)
+
```

Comment:
Again, we can add tonnes of different datasets - why clinical? In that case we could add all sorts of specialised expert data. I know you guys are into your health data, but I'm not sure if it makes sense to include it in a test set. If we _do_ want to include such data then we could bulk it with expert data from other fields (law, finance, Danish research articles etc., but since we don't have this atm, I'd leave it for future work).

Reply:
I think two of the main application areas in industry at the moment are ClinicalNLP and LegalNLP. I think it is fair to build specifically into this. Especially as I think the test set should be actionable, that is that a practitioner should be able to decide whether it is worth using the model for their task given the performance in their domain. That being said @HLasse and I plan to do a domain-specific fine-tuning of a DFM model for The Psych. domain so it can def. be left for that paper.

Reply:
Ah right, fair enough. But sure, having two expert-style texts could make sense, I was just worried that the same argument could be made for every specialised industry, but I get your point in including the two most popular ones. I'll update that.